.gitignore

```diff
@@ -1,3 +1,6 @@
-custom/config.yml
-custom/people.opml
-custom/people.opml.bak
+/admin/inc/pwd.inc.php
+/cache
+/custom/config.yml
+/custom/people.opml
+/custom/people.opml.bak
+/nbproject/private/
```

Comment:
It's an subfolder created and used by Netbeans: this allows all the users of this IDE to open the project without having to import it.